domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RawEntity.java

```diff
@@ -18,24 +18,44 @@
 
 import java.util.Map;
 
+/**
+ * The {@code RawEntity} class represents a row of a GTFS file as raw string data.
```

Comment:
From the recommendations given at: https://blog.joda.org/2012/11/javadoc-coding-standards.html; I understood that it is better not using @code in the first sentences. 

I modify accordingly to your advice.

Reply:
Actually, here, you're right (I missed that this was referring the same class we're already in). So this can stay as `@code`.

And you're right regarding the article, although I'm not sure I agree with it. Generally, I think if the class is mentioned more than once then follow the article advice. If it's only mentioned once, then I would prefer `@link` instead of `@code` to be able to jump to that class easily.